schema-registry/rest-service/src/test/java/com/hortonworks/registries/schemaregistry/avro/util/CustomParameterizedBlockJUnit4Runner.java

```diff
@@ -0,0 +1,242 @@
+/**
```

Comment:
I am not sure about licensing issues with EPL when we copy the code in ASL based sources like here. You may need to put their license(EPL) here and other classes where junit's code is copied.